dist/profile/templates/trusted.ci.jenkins.io/agents.yaml.erb

```diff
@@ -0,0 +1,23 @@
+---
+jenkins:
+  ## No builds on master
+  numExecutors: 0
+  agentProtocols:
+  - "JNLP4-connect"
+  - "Ping"
+  nodes:
+  - permanent:
+      labelString: "updatecenter census"
+      launcher:
+        ssh:
+          credentialsId: "0e73a07a-5f72-4a90-bcce-5dcfcd10aea8"
```

Comment:
ideally rename the cred to something that makes sense.

Reply:
Yeah, but not part of this PR though (the goal here is to turn the switch on). I thought about managing credentials directly from CasC to solve this issue.